src/common/uppy.utils.tsx

```diff
@@ -13,7 +13,7 @@ export const StyledUppyBox = styled(Box)(({ theme }) => ({
     maxHeight: 'calc(100% - 64px)',
     width: 'calc(100% - 64px)',
   },
-  '& .uppy-Dashboard--modal .uppy-Dashboard-inner': { zIndex: 1300 },
+  '& .uppy-Dashboard--modal .uppy-Dashboard-inner': { zIndex: 1300 + 1 },
```

Comment:
I think this could do with a comment on what component has the value of 1300 so we know to update it if it changes.